File: integration-tests/src/test/java/stubidp/test/integration/support/TestStubIdp.java
package stubidp.test.integration.support;

import stubidp.stubidp.configuration.StubIdp;
import stubidp.stubidp.configuration.UserCredentials;

import java.util.List;

public class TestStubIdp extends StubIdp {

    public TestStubIdp(String assetId, String displayName, String friendlyId, List<UserCredentials> idpUserCredentials, boolean sendKeyInfo) {
        this.sendKeyInfo = sendKeyInfo;
        this.assetId = assetId;
        this.displayName = displayName;
        this.friendlyId = friendlyId;
        this.idpUserCredentials = idpUserCredentials;
    }

}

File: integration-tests/src/test/kotlin/stubidp/kotlin/test/integration/support/TestStubIdp.kt
package stubidp.kotlin.test.integration.support

import stubidp.stubidp.configuration.StubIdp
import stubidp.stubidp.configuration.UserCredentials

class TestStubIdp(assetId: String?, displayName: String?, friendlyId: String?, idpUserCredentials: List<UserCredentials?>?, sendKeyInfo: Boolean) : StubIdp() {
    init {
        this.sendKeyInfo = sendKeyInfo
        this.assetId = assetId
        this.displayName = displayName
        this.friendlyId = friendlyId
        this.idpUserCredentials = idpUserCredentials
    }
}